src/Raven.Studio/typescript/viewmodels/manage/adminLogs.ts

```diff
@@ -210,6 +213,12 @@ class adminLogs extends viewModelBase {
             
             this.duringManualScrollEvent = false;
         });
+      
+        $( ".list-view" ).on( "click", "a", function(event) {
+            event.preventDefault();
+            event.stopImmediatePropagation();
+            copyToClipboard.copy($(this).parent().children().last().text(), "Log message has been copied to clipboard");
```

Comment:
I think we should use better selector  - this is really fragile

Reply:
now using: `$(this).next().text()`

Reply:
can we use $("span", this).text() ?

Reply:
The dom structure per item is:

```
<div>
    <pre>
        <a>the copy button</a>
        <span>the text to copy</span>
    </pre>
</div>
```

`$(this)` is the `<a>` element that was clicked on - it has no text on it.
so I go to `$(this).next()` - which is the `<span>` that has the text.

Reply:
oh ok, so `$(this).next().text()` should be good